src/Adapter/Cart/QueryHandler/GetCartForViewingHandler.php

```diff
@@ -176,7 +176,7 @@ public function handle(GetCartForViewing $query)
             'total_shipping_formatted' => Tools::displayPrice($total_shipping, $currency),
             'total' => $total_price,
             'total_formatted' => Tools::displayPrice($total_price, $currency),
-            'is_tax_included' => $tax_calculation_method === PS_TAX_INC,
+            'is_tax_included' => $tax_calculation_method == PS_TAX_INC,
```

Comment:
😭 
@sarjon I guess types do not match ?

Reply:
Yeah, legacy takes it from db and it can result in `string`, so I've kept legacy behavior here.